src/trigger/importProject/index.ts

```diff
@@ -0,0 +1,308 @@
+import { updateDocumentMetadata } from '@backend/crud';
+import { getDownloadURL } from '@backend/storage';
+import { createClient, PostgrestError, type SupabaseClient } from '@supabase/supabase-js';
+import { logger, task } from '@trigger.dev/sdk/v3';
+import { generatePassword } from '@util/auth';
+import AdmZip from 'adm-zip';
+import { v4 as uuidv4 } from 'uuid';
+
+interface Payload {
+  jobId: string;
+  token: string;
+}
+
+const IIIF_KEY = process.env.IIIF_KEY || process.env.RECOGITO_TIGER;
```

Comment:
I see this `RECOGITO_TIGER` somewhere else too, any idea what it is? (if not, can probably be removed from both places)

Reply:
I have no idea what that is... the other place is in the API handler that forwards the image uploads to IIIF cloud. But I don't think I put it there. Looks like it came in with [this commit](https://github.com/recogito/recogito-client/commit/2112eb022eb73ec8d3400e9c8bcac824818cdca9).